### PR TITLE
ci: check the license agreement on Travis.

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
Some of this checks might not be needed.
For example, if the PR is from a Canonical employee and he's not using his canonical email address, then it's ok to fail. He should probably change his git configuration to send code identifying himself as a canonicaler.

Then this could be reduced to two cases:
- a canonical address: good
- member of the launchpad cla team: good

All the rest could be considered as failures. Let me know what do you think, and I you prefer I'll remove the other two cases.

File: tools/cla_check_in_travis.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    # We assume that the members of the snapcore team have signed the agreement.
+    echo 'Branch from a snapcore team member. CLA OK.'
+    exit 0
+elif [ "$(echo $email | cut -d @ -f 2)" = 'canonical.com' ]; then
+    echo 'Branch from a Canonical employee. CLA OK.'
+    exit 0
+elif ./tools/cla_check_in_launchpad.py $email; then
+    echo 'Branch from a signed contributor. CLA OK.'
+    exit 0
+elif ./tools/past_contributor_check.py $owner; then
```
Comment (sergiusens) [inline tools/cla_check_in_travis.sh]: I'd rather not do this last else

Comment (elopio) [inline tools/cla_check_in_travis.sh]: ack. Then I will leave only the two cases mentioned above.

**Comment by elopio** [comment]
@sergiusens do you want me to keep on working on the script to check if it's a member of the canonical team in launchpad, or do you think it's enough to check the email address?

**Comment by sergiusens** [comment]
@elopio if we check for anything we should check that if the submitter is a member of canonical they should submit the PR/commits with the canonical address

File: tools/cla_check_in_travis.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ev
+
+email="$(git log -1 $TRAVIS_COMMIT --pretty="%aE")"
+echo "Checking the licence agreement for $email..."
```
Comment (kyrofa) [inline tools/cla_check_in_travis.sh]: s/licence/license/

Comment (elopio) [inline tools/cla_check_in_travis.sh]: @kyrofa I copied it from the website. But now I see it's used inconsistently, so I reported a bug:
https://github.com/ubuntudesign/www.ubuntu.com/issues/1200

I think we should use licence, even though it looks ugly.

Comment (kyrofa) [inline tools/cla_check_in_travis.sh]: Ah, british spelling. I thought it was a typo :stuck_out_tongue: .

**Comment by sergiusens** [comment]
Mind fixing the conflicts?

**Pull request merged**